<?xml version="1.0" encoding="utf-8"?>

<launch>

	<rosparam file="$(find grasp_learning)/param/PGR_3.yaml" command="load"/>
		<node name="PGR_3" pkg="grasp_learning" type="PGR_3" respawn="false" output="screen"/>

</launch>